promgen/templates/promgen/alert_list.html

```diff
@@ -20,8 +20,8 @@
         <th>Service</th>
         <th>Project</th>
         <th>Job</th>
-        <th>Sent</th>
-        <th>Error</th>
+        <th><a href="?sent=0">Sent</a></th>
+        <th><a href="?error=1">Error</a></th>
```

Comment:
Why `sent` is set to `0` and `error` is set to `1`?

Reply:
Partly I was thinking of `zero sent emails` or `at least one error` but it probably does not matter significantly since we're only doing `filter()` or `exclude()` on the backend and not actually looking at the value.

Reply:
Do you have a better recommendation for a value to use for these ? I could also change the query parameter name.

Reply:
In both cases we are enabling a filter to only display alerts sent or alerts with errors, so I guess having `1` in both of them (`1` meaning `true`) looks better.

However, since we are not looking at the value, if the user manually changes the value and sets a `0` to disable the filter, it will not work as expected. So, I guess it would be better to look at the value.

Also, if you click on those links, the page reloads with a different url containing the new key, with the filter enabled. But, if I'm not mistaken, those links will appear again in the same way. What happens if you click again on the same one that you clicked before? I would expect the filter to be disabled. Is that the current behavior?

Reply:
Right now, none of the other search filters (service,project,etc) have a toggle either, so for now I'm just copying the behavior.

Reply:
I think the naming is too picky, but I've changed it in 6d9030e
`noSent` and `sentError` so they're more like flags.

Reply:
I guess it's fine by now.